packages/wordpress-kit/README.md

```diff
@@ -28,10 +30,31 @@ const client = new GraphqlClientFactory(
 ).create();
 ```
 
+### TailwindcssPlugin
+
+1. Import the plugin to your `tailwind.config.js` file
+
+```
+const { tailwindcssPlugin } = require("@pantheon-systems/wordpress-kit");
+```
+
+2. Add the plugin to your `plugins` array:
+
+```
+module.exports = {
+  content: [ ... ],
+  theme: { ... },
+  plugins: [tailwindcssPlugin],
+};
+```
+
+3. (Optional) Extend the plugin or override the theme in the `theme.extend`
```

Comment:
Probably can be helpful add this link to the tailwind docs where that's explained: https://tailwindcss.com/docs/theme#extending-the-default-theme